independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/ListCategoriesCommandHandler.java

```diff
@@ -54,27 +53,12 @@ public QuarkusCommandOutcome execute(QuarkusCommandInvocation invocation) throws
         final BuildTool buildTool = invocation.getQuarkusProject().getBuildTool();
         boolean isGradle = BuildTool.GRADLE.equals(buildTool) || BuildTool.GRADLE_KOTLIN_DSL.equals(buildTool);
 
-        if (!cli) {
-            if (StringUtils.isBlank(format)) {
-                if (isGradle) {
-                    log.info(ListCategories.MORE_INFO_HINT, "--format=full");
-                } else {
-                    log.info(ListCategories.MORE_INFO_HINT, "-Dquarkus.extension.format=full");
-                }
-            }
-
-            if (isGradle) {
-                log.info(ListCategories.LIST_EXTENSIONS_HINT,
-                        "`./gradlew listExtensions --category=\"categoryId\"`");
-            } else {
-                log.info(ListCategories.LIST_EXTENSIONS_HINT,
-                        "`./mvnw quarkus:list-extensions -Dcategory=\"categoryId\"`");
-            }
-        } else if (!batchMode) {
-            if (StringUtils.isBlank(format)) {
+        if (cli && !batchMode) {
+            if ("name".equalsIgnoreCase(format)) {
```

Comment:
This last hint should move into the CLI itself (emitted before it calls the command... be wary (given now updated code to list extensions for an arbitrary version) that we change the default value for that case to use "origins" (since showing where the list is coming from is useful).

Reply:
@ebullient i.e. you want to have the format hint printed before the list or other hints too?